usage/jsgui/src/main/webapp/assets/js/view/application-add-wizard.js

```diff
@@ -105,15 +109,15 @@ define([
                               step_id:'what-app',
                               title:'Create Application',
                               instructions:'Choose or build the application to deploy',
-                              view:new ModalWizard.StepCreate({ model:this.model, wizard: this })
+                              view:new ModalWizard.StepCreate({ model:this.model, wizard: this, catalog: this.catalog })
                           },
                           {
                               // TODO rather than make this another step -- since we now on preview revert to the yaml tab
                               // this should probably be shown in the catalog tab, replacing the other contents.
                               step_id:'name-and-locations',
                               title:'<%= appName %>',
-                              instructions:'Specify the locations to deploy to and any additional configuration',
-                              view:new ModalWizard.StepDeploy({ model:this.model })
+                              instructions:'Specify the application version, locations to deploy to and any additional configuration',
```

Comment:
it's not the version of the actual *application* -- it's the version of the blueprint -- so this wording would be confusing.

i suggest leave as it was, simplify -- we don't always show version, and if we do it's obvious enough what it is.  in time we could have a hover-help if it's an issue.